Feat/onramp (#1412)
WalletConnect/walletconnect-docs: docs/web3modal/shared/onramp.mdx

---
title: onRamp
---

> Web3Modal SDK integrates Coinbase Pay, which lets users sign-in to their Coinbase account and use their saved payment methods or crypto balance on self-custody wallets or dapps.

Reviewer: It's not like they use their fiat methods on wallets or dapps but rather buy crypto with them. 
@Sam-Newman wyt?

Author: isn't it both? I also took this from https://docs.cloud.coinbase.com/pay-sdk/docs/intro

Author: ```suggestion
Web3Modal SDK integrates Coinbase Pay, which lets users sign-in to their Coinbase account and use fiat payment methods to purchase cryptocurrencies.
```

What do you think?

Author: updated, thanks!

Reviewer: This is better imo!
